### DISCUSSION TITLE
Thanks for all and let's help Panchovix

### DISCUSSION THREAD

**Discussion by Bullseye110** [Panchovix/stable-diffusion-webui-reForge#63] [Ideas]
Hello!
First thing first, thanks for continuing the development of a good project called Forge and abandoned by its main developer.
The best thing will be if more devs/coders will help you at making this ReForge tool the best out there. Unfortunately, I'm no coder/dev. But the real devs/coders who are sad that SD Forge was abandoned can help Panchovix here with things he doesn't know and with more ideas.
This field can evolve only when open minded people unite for achieving greater goals.
I have a proposal for Panchovix - you can start a Patreon/buymeacoffee and the people who appreciate your work will contribute how they can.
Congrats for this and good luck! Keep up the good work!
And do not forget! The open-source community can beat pay2use companies/politics only by uniting. There's no other way around as being alone in this domain is overwhelming.

**Comment by Panchovix** [comment]
Many thanks for your post, I really, really appreciate it!

I did post a buymeacoffee link here! https://www.buymeacoffee.com/Panchovix

Also as you mention, this have been more work that I expected, and this mixed with my IRL work is a bit hard to implement the harder features missing (for example, on dev_upstream, adding the new models like SD3, SD Cascade, Aura Flow etc into forge_loader.py, all the code to make these models work are already on the backend!)

With a bit of help I think we can manage that and a lot of more, so any help is really welcome! Also many thanks for people doing suggestions and bug reports, since there I can improve and fix issues in the repo.

Again, many thanks for the good vibes and the post!

EDIT: Also many thanks to the 5EUR Donation on paypal before the change!

**Reply by Bullseye110** [reply]
You're welcome and yea, make a Patreon/Buymeacoffee as some people prefer to not share their real email address like on PayPal.

**Reply by Panchovix** [reply]
Many thanks for the suggestion, then, did a buymeacoffe here! https://www.buymeacoffee.com/Panchovix

Gonna update the readme!
